app/models/Metadata.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+
+class Metadata():
+    def __init__(self, is_active: bool = True, created_at: str = time.time(), updated_at: str = time.time()):
```

Comment:
do we really need all this complexity? can we just use Pydantic model with public fields?